README.md

```diff
@@ -49,6 +49,7 @@ List of content:
 * [Staketab pool](https://staketab.com/) A staking pool with 7% fee
 * [ZKValidator pool](https://zkvalidator.com/) A staking pool with 10% fee
 * [P2P Validator](https://p2p.org/) A staking pool with 8% fee
+* [0base.vc pool](https://mina.0base.vc) A beginner's luck strategy applied staking pool with 5% fee
```

Comment:
Since staking pools might end up turning controversial, and in order to prevent a spammy promotional board, I would suggest to everybody stick to the same format `name + fees` 🙏